### ISSUE TITLE
Decide which units we will measure position in, and also decide how fast impulse and warp is

### ISSUE THREAD

**Issue by jextrevor** [X-12/andromeda#92]
I vote for using AU for the position. Warp 1 would equal light speed (converted into AU's/second) and 100 impulse would equal half of light speed.

What do you guys think?

**Comment by jextrevor** [comment]
We also ought to discuss more about how we would like the position system to work and how we want the thrusters to work.

**Comment by jextrevor** [comment]
Do you have any objections to using AU's, as putting the speed for Warp 1 to light speed, and as putting the speed for 100 impulse (100%) to, say, 1 time around the earth per minute?

**Comment by caydenlund** [comment]
It might be better to use the metric system.

**Comment by jextrevor** [comment]
I agree but I'm kind of scared of how large values will get using meters. And we can still use prefixes with AU, e.g. kAU, cAU, pAU

**Comment by caydenlund** [comment]
We can use prefixes with the metric system. Gigameters and whatnot. That's
what I would prefer, but I'll leave this one up to you.

**Comment by jextrevor** [comment]
Yottametres. And no, we need to decide this as a group
